### ISSUE TITLE
Add codeforces Global Round 16 , 12sep 2021

### ISSUE THREAD

**Issue by theparasgami** [saumyasrivastava08/Important_Coding_Problems#25]
Hey,
I would like to add Codeforces Global round 16 which was held on 12 Sep 2021. 
Requesting to be assigned to this issue under the 'hacktoberfest2021' label. Thanks.

**Comment by saumyasrivastava08** [comment]
> Hey, I would like to add Codeforces Global round 16 which was held on 12 Sep 2021. Requesting to be assigned to this issue under the 'hacktoberfest2021' label. Thanks.

Yeah Sure Name the folder as contest name . Add the contest link + problem link.

**Comment by theparasgami** [comment]
Okay,  Thanks.

**Comment by navjotdadwal** [comment]
sir can you please assign this issue to me i am current rated 5 star at codechef with rating of 2153 and rated expert at codeforces with rating of 1776 and i use cpp language for competitive programming and can resolve this issue
following are my accounts:
https://codeforces.com/profile/navjotdadwal
https://www.codechef.com/users/navjot1234
